docs/pipelines.md

```diff
@@ -143,28 +147,31 @@ For `cohortextractor run` to work:
 
 To run the first action in the example above, using dummy data, you can use:
 
-```sh
-cohortextractor run dummy generate_study_population expectations
+```bash
+opensafely run generate_study_population
 ```
 
 This will generate the `input.csv` file as explained in the [cohortextractor](cohortextractor.md) section.
 
 To run the second action you can use:
-```sh
-cohortextractor run dummy run_model expectations
+
+```bash
+opensafely run run_model
 ```
-As this action depends on the `generate_study_population` action, it will cause `generate_study_population` to be run first, followed by `run_model`.
+
 It will create the two files as specified in the `analysis/model.do` script.
 
-To force the dependencies to be run you can use for example `cohortextractor run run_model --force-run --force-dependencies`. This will ensure that both the `run_model` and `generate_study_population` actions are run, even if `input.csv` already exists.
+To force the dependencies to be run you can use for example `opensafely run run_model -f`. 
```

Comment:
I'd be tempted to use the long form here, i.e. `--force-run-dependencies`) and optionally say "`-f` for short" if you want